hack/jenkins/e2e.sh

```diff
@@ -85,12 +170,20 @@ export KUBE_AWS_ZONE=${E2E_ZONE}
 export INSTANCE_PREFIX=${E2E_CLUSTER_NAME}
 export KUBE_GCE_ZONE=${E2E_ZONE}
 export KUBE_GCE_NETWORK=${E2E_NETWORK}
+export KUBE_GCE_INSTANCE_PREFIX=${KUBE_GCE_INSTANCE_PREFIX:-}
+export KUBE_GCS_STAGING_PATH_SUFFIX=${KUBE_GCS_STAGING_PATH_SUFFIX:-}
 
 # GKE variables
 export CLUSTER_NAME=${E2E_CLUSTER_NAME}
 export ZONE=${E2E_ZONE}
 export KUBE_GKE_NETWORK=${E2E_NETWORK}
 
+# Shared cluster variables
+export MASTER_SIZE=${MASTER_SIZE:-}
+export MINION_SIZE=${MINION_SIZE:-}
+export NUM_MINIONS=${NUM_MINIONS:-}
+export PROJECT=${PROJECT:-}
+
```

Comment:
+1 to just dumping env (maybe after everything is set (e.g. line 196 ish)) before tests start